source/server/http/admin.cc

```diff
@@ -128,6 +147,7 @@ Http::Code AdminImpl::handlerClusters(const std::string&, Buffer::Instance& resp
           fmt::format("{}::{}::canary::{}\n", cluster.second->name(), host->url(), host->canary()));
       response.add("\n");
     }
+    response.add("\n");
```

Comment:
I updated so that its intention is clear and does not cause double newlines when there are actually hosts.